### PR TITLE
New application: lhvy.pipes-rs version 1.6.0

### PR DESCRIPTION
**Pull request by KaranKad**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

```
winget install -m .\1.6.0\
Found pipes-rs [lhvy.pipes-rs] Version 1.6.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/lhvy/pipes-rs/releases/download/v1.6.0/pipes-rs-windows-x86_64.zip
  ██████████████████████████████   471 KB /  471 KB
Successfully verified installer hash
An anti-virus product reports an infection in the installer
```

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95428)

### CODE REVIEW

**Comment by KaranKad** [comment]
https://www.virustotal.com/gui/file/38dfeaad55832dc37a770af1c5c0079055b2648f6308236130dea91f274c9ff1/detection

File: manifests/l/lhvy/pipes-rs/1.6.0/lhvy.pipes-rs.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+PackageIdentifier: lhvy.pipes-rs
+PackageVersion: 1.6.0
+PackageLocale: en-US
+Publisher: lhvy
+Author: Lucas
+PackageName: pipes-rs
+PackageUrl: https://github.com/lhvy/pipes-rs
+License: Apache-2.0, MIT
+LicenseUrl: https://raw.githubusercontent.com/lhvy/pipes-rs/master/LICENSE-APACHE, https://raw.githubusercontent.com/lhvy/pipes-rs/master/LICENSE-MIT
```
Comment (stephengillie) [inline manifests/l/lhvy/pipes-rs/1.6.0/lhvy.pipes-rs.locale.en-US.yaml:12]:
```suggestion
LicenseUrl: https://raw.githubusercontent.com/lhvy/pipes-rs/master/LICENSE-APACHE ; https://raw.githubusercontent.com/lhvy/pipes-rs/master/LICENSE-MIT

```

Comment (stephengillie) [reply manifests/l/lhvy/pipes-rs/1.6.0/lhvy.pipes-rs.locale.en-US.yaml:12]: Adding a space to separate the delineator from the URI.

File: manifests/l/lhvy/pipes-rs/1.6.0/lhvy.pipes-rs.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+PackageIdentifier: lhvy.pipes-rs
+PackageVersion: 1.6.0
+InstallerLocale: en-US
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: pipes-rs.exe
+  PortableCommandAlias: pipes-rs
+Installers:
```
Comment (stephengillie) [inline manifests/l/lhvy/pipes-rs/1.6.0/lhvy.pipes-rs.installer.yaml:12]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Comment (stephengillie) [reply manifests/l/lhvy/pipes-rs/1.6.0/lhvy.pipes-rs.installer.yaml:12]: The package installs, but requires the dependency to run:
image

**Comment by stephengillie** [comment]
@KaranKad,

Apologies, I mistakenly thought we had added dependency support.

**Comment by KaranKad** [comment]
No issue, we will wait.

**Review by ItzLevvie — APPROVED** [review]

File: manifests/l/lhvy/pipes-rs/1.6.0/lhvy.pipes-rs.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+PackageIdentifier: lhvy.pipes-rs
+PackageVersion: 1.6.0
+PackageLocale: en-US
+Publisher: lhvy
+Author: Lucas
+PackageName: pipes-rs
+PackageUrl: https://github.com/lhvy/pipes-rs
+License: Apache-2.0, MIT
+LicenseUrl: https://raw.githubusercontent.com/lhvy/pipes-rs/master/LICENSE-APACHE , https://raw.githubusercontent.com/lhvy/pipes-rs/master/LICENSE-MIT
```
Comment (stephengillie) [inline manifests/l/lhvy/pipes-rs/1.6.0/lhvy.pipes-rs.locale.en-US.yaml:12]:
```suggestion
LicenseUrl: https://raw.githubusercontent.com/lhvy/pipes-rs/master/LICENSE.md
```

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**